packages/compass-import-export/config/webpack.base.config.js

```diff
@@ -73,7 +73,7 @@ module.exports = {
       test: /\.node$/,
       use: 'node-loader'
     }, {
-      test: /node_modules[\\\/]JSONStream[\\\/]index\.js/,
+      test: /node_modules[\\/]JSONStream[\\/]index\.js/,
```

Comment:
@mcasimir is this ok? linter gave me error as un-necessary `\`

Reply:
The "auto-fix" lamp in VSCode should do the right thing there. Though that's a bit strange, it doesn't complain in other packages.